src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,78 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+  const [name, setName] = useState<string>(undefined);
+  const [subject, setSubject] = useState<string>(undefined);
+  const [content, setContent] = useState<string>(undefined);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (!name) {
+      setName("");
+      return;
+    }
+    if (!subject) {
+      setSubject("");
+      return;
+    }
+    if (!content) {
+      setContent("");
+      return;
+    }
+
+    const body =
+      `Hello,\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Yours sincerely,\n` +
+      `${name}\n`;
+
+    location.href =
```

Comment:
Can explain the purpose of `location` interface ?

Reply:
I think it's to set the current URL

Reply:
hem not url doank I think dia bisa change port, protocol, etc jg